src/main/java/com/google/devtools/build/lib/rules/java/JavaRuntimeRule.java

```diff
@@ -83,6 +84,10 @@ public RuleClass build(RuleClass.Builder builder, RuleDefinitionEnvironment env)
         <!-- #END_BLAZE_RULE.ATTRIBUTE --> */
         .add(attr("java_home", STRING))
         .add(attr("output_licenses", LICENSE))
+        /* <!-- #BLAZE_RULE(java_runtime).ATTRIBUTE(version) -->
+        The major version of the Java runtime.
```

Comment:
Can this documentation mention the version number here is the same as the 'feature element of the version number' documented in https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/lang/Runtime.Version.html#feature() ?